packages/test/test-end-to-end-tests/src/test/summaries.spec.ts

```diff
@@ -319,3 +358,204 @@ describeNoCompat("Summaries", (getTestObjectProvider) => {
+        const submitResult3 = await result3.summarySubmitted;
+        assert(submitResult3.success, "on-demand summary3 should submit");
+        assert(submitResult3.data.stage === "submit",
+            "on-demand summary3 submitted data stage should be submit");
+
+        const broadcastResult3 = await result3.summaryOpBroadcasted;
+        assert(broadcastResult3.success, "summary op3 should be broadcast");
+        const summary3ParentHandle = broadcastResult3.data.summarizeOp.contents.head;
+        assert(summary3ParentHandle === summary1AckHandle, "Summary Parent should match ack handle of summary1");
+    });
+
+    it("Single commit summary/Match last summary ackHandle  with current summary parent", async function() {
+        if (provider.driver.type !== "odsp") {
+            this.skip();
+        }
+
+        const { mainContainer, summarizer } = await createMainContainerAndSummarizer(provider,
+            configForSingleCommitSummary);
+
+        // Summarize
+        const result: ISummarizeResults = summarizer.summarizeOnDemand({ reason: "test" });
+        const submitResult = await result.summarySubmitted;
+        assert(submitResult.success, "on-demand summary should submit");
+        assert(submitResult.data.stage === "submit",
+            "on-demand summary submitted data stage should be submit");
+
+        const broadcastResult = await result.summaryOpBroadcasted;
+        assert(broadcastResult.success, "summary op should be broadcast");
+        const summary1ProposedHandle = broadcastResult.data.summarizeOp.contents.handle;
+
+        const ackNackResult = await result.receivedSummaryAckOrNack;
+        assert(ackNackResult.success, "summary op should be acked");
+        const summary1AckHandle = ackNackResult.data.summaryAckOp.contents.handle;
+        assert(summary1ProposedHandle === summary1AckHandle, "Summary proposed and ack handle should match");
+        summarizer.close();
+
+        const summarizer2 = await createSummarizerFromContainer(provider, mainContainer,
+            configForSingleCommitSummary);
+        // Second Summary
+        const result2: ISummarizeResults = summarizer2.summarizeOnDemand({ reason: "test2" });
+        const submitResult2 = await result2.summarySubmitted;
+        assert(submitResult2.success, "on-demand summary2 should submit");
+        assert(submitResult2.data.stage === "submit",
+            "on-demand summary2 submitted data stage should be submit");
+
+        const broadcastResult2 = await result2.summaryOpBroadcasted;
+        assert(broadcastResult2.success, "summary op2 should be broadcast");
+        const summary2ParentHandle = broadcastResult2.data.summarizeOp.contents.head;
+        assert(summary2ParentHandle === summary1AckHandle,
+            "Summary Parent should match ack handle of previous summary");
+    });
+
+    it.skip("Single commit summary/Last summary should not be discarded due to missing SummaryOp", async function() {
```

Comment:
>skip( [](http://example.com/codeflow?start=7&length=5)

why are we skipping this one ?

Reply:
Because this is not passing due to the underlying issue for single commit summary. I will fix that and enable this test.